docs/logs/complex-objects/complex-objects.csproj

```diff
@@ -1,6 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <SuppressTfmSupportBuildWarnings>true</SuppressTfmSupportBuildWarnings>
+    <EmitCompilerGeneratedFiles>true</EmitCompilerGeneratedFiles>
```

Comment:
This would put the generated source under `.\obj\Debug\net8.0\generated\Microsoft.Gen.Logging\Microsoft.Gen.Logging.LoggingGenerator\Logging.g.cs` which helps to understand things behind the scene.

Reply:
Just FYI in case you weren't aware you can see them in IDE under the analyzer:

![image](https://github.com/open-telemetry/opentelemetry-dotnet/assets/28367120/efa2477e-0907-427b-a07e-15072bc5e807)